README.md

```
@@ -50,17 +58,35 @@ A proxy server that enables Cursor IDE to use GitHub Copilot's API instead of Cu
+   npm run start:verbose
+   ```
+
 ## 🔌 Configuration with Cursor IDE
 
 1. Open Cursor IDE
 2. Go to Settings > API Keys
 3. In the "Override OpenAI Base URL" section, enter:
    ```
-   http://localhost:3000
+   http://localhost:3000/v1
    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)